### DISCUSSION TITLE
bug: inconsistent state between achievements page and profile section

### DISCUSSION THREAD

**Discussion by vladdoster** [community/community#39400] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The badge is initially hidden. I select the "show option", but it isn't reflected. This seems to make any further changes to be flip-flopped.

It seems the backend state is correct, but this behavior seems like a frontend state bug.

https://user-images.githubusercontent.com/10052309/202642265-ccad3118-a7f8-43c9-998b-c36716677bfd.mov

**Comment by radar-blip** [comment]
You're probably just clicking on it to many times, to fast. Switch to decaf 😄

**Comment by vladdoster** [comment]
@linuxlawson I don't think so. I can reproduce it in Safari and Vivaldi and log in via a private/incognito window.

**Reply by radar-blip** [reply]
You're right, it seems a bit buggy.  There's a delay until you reload the page.
